src/meta/model_v2/src/table.rs

```diff
@@ -108,7 +108,6 @@ pub struct Model {
     pub distribution_key: I32Array,
     pub stream_key: I32Array,
     pub append_only: bool,
-    pub properties: Property,
```

Comment:
Need to change definition here: https://github.com/risingwavelabs/risingwave/blob/c8030fc0f2179c563452afb1983411a70267c455/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L1013